node/src/blockchain/blockchain_interface.rs

```diff
@@ -404,7 +416,7 @@ where
         }
     }
 
-    fn log_debug_before_sending(&self, inputs: &SendTransactionInputs) -> String {
+    fn before_sending_msg(&self, inputs: &SendTransactionInputs) -> String {
```

Comment:
Maybe this should be called a preparation log, and the next one should be called a transmission log..